### ISSUE TITLE
Can support ldaps with port 636?

### ISSUE THREAD

**Issue by conan522** [thorsten/phpMyFAQ#3210]
Can support ldaps with port 636? Looks does not support ldaps now.

**Comment by thorsten** [comment]
LDAPS is supported, you just have to configure it correctly in the ldap.php configuration file, e.g.:

`<?php
// Main LDAP server
$PMF_LDAP['ldap_server'] = 'ldaps://192.168.2.163';
$PMF_LDAP['ldap_port'] = 636;
$PMF_LDAP['ldap_user'] = 'cn=admin,dc=phpmyfaq,dc=com';
$PMF_LDAP['ldap_password'] = 'Password';
$PMF_LDAP['ldap_base'] = 'ou=people,dc=phpmyfaq,dc=com';`

The file has to be in phpmyfaq/config/ldap.php if you use phpMyFAQ 3.2 or earlier.

**Comment by conan522** [comment]
I installed 3.2.9. How can add a domain account in "user mgmt"? which format: domain/username or email. neither can not find useres.

**Comment by thorsten** [comment]
You can edit the LDAP configuration under Configuration->LDAP, there standard values as default values.

**Comment by conan522** [comment]
I did LDAP configuration as  standard. But how can I test is it works. Can we add a domain user in "User Mgmt" or login with a domain user? Both did not work for me. So far I have no idea LDAP is works or not.

**Comment by thorsten** [comment]
Just login with your LDAP user and it should work.

**Comment by conan522** [comment]
error message is "Wrong username or password." Also what is the username format: "domain/username" or "email address". But neither them got same error message. BTW, if do not need add LDAP user in "Users" in admin, how can grant permission to which user can access FAQS if I do not any visitor can access . It is does not make sense. Thanks.

**Comment by thorsten** [comment]
With the first successful login the LDAP user will be stored in the phpMyFAQ database and when you can add the permissions. For the LDAP questions you have to ask your LDAP admin in your company as I'm not a LDAP expert. Sorry. Usually you should see the error in your LDAP server logs.

**Comment by conan522** [comment]
Yes, I will check LDAP server logs and ask LDAP admin if it is a LDAP issue. But could you tell me how to fill in ldap username in login field. Is "domain/username" or "username" or "email address"? which is correct format? Thanks.

**Comment by thorsten** [comment]
@conan522 depends on your configuration

**Comment by conan522** [comment]
sorry，could you share where is the configuration and how to setup? Thanks.

**Comment by thorsten** [comment]
Just log into the admin backend, go to Configuration and open the LDAP tab. There you'll find all the LDAP configuration items.

**Comment by thorsten** [comment]
@conan522 any feedback on this?

**Comment by thorsten** [comment]
no feedback -> closed
